sys-cluster/ganglia-web/ganglia-web-3.7.3.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+WEBAPP_MANUAL_SLOT="yes"
+inherit webapp eutils
+
+DESCRIPTION="Web frontend for sys-cluster/ganglia"
+HOMEPAGE="http://ganglia.sourceforge.net"
+SRC_URI="https://github.com/ganglia/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE="vhosts"
+
+DEPEND="net-misc/rsync"
+RDEPEND="
+	${DEPEND}
+	${WEBAPP_DEPEND}
+	>=sys-cluster/ganglia-3.3.7[-minimal]
+	dev-lang/php[gd,xml,ctype]
+	virtual/httpd-php
+	media-fonts/dejavu"
+
+src_configure() {
+	return 0
+}
+
+src_compile() {
```

Comment:
why?(2). Just empty function, I do not see the point for this to be here as the previous one